spec/extensions/hash_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require_relative "../rspec_helper"
+
+RSpec.describe "Hash" do
```

Comment:
Most of the tests here are for methods that are now implemented by ActiveSupport.  I don't think we need to include tests for the functionality we're getting from gems.